datastore/integration_test.go

```diff
@@ -811,6 +811,10 @@ func TestIntegration_AggregationQueriesInTransaction(t *testing.T) {
 			if readTime.IsZero() {
 				// Use current time as read time if read time is not specified in test case
 				readTime = time.Now().Truncate(time.Microsecond)
+
+				// Read time is truncated to microseconds. If immediately used in NewTransaction call,
+				// it leads to "read_time cannot be in the future" error
+				time.Sleep(time.Second)
```

Comment:
Should the sleep time here be "Microsecond" or "Second" since the read time is truncated to microseconds?

Reply:
It can be microsecond but being safe here to prevent failures if and when deviation between service clock and test runner clock is more